src/libraries/Native/Unix/verify-entrypoints.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+dllList=$(nm $1 | awk '/^[0-9a-fA-F]+ T \S+/ { if ($3 != "_init" && $3 != "_fini") print $3 }')
+entriesList=$(awk 'match($0, /^\s+DllImportEntry\((\S+)\)/, m) { print m[1]}' $2)
```

Comment:
This expression only worked for me on macOS but failed on Linux, FreeBSD and illumos. The following variant works on all (on illumos/OpenIndiana, worked with nawk but not with the default; oawk):

```sh
entriesList=$(awk -F'DllImportEntry\\(|\\)' '/DllImportEntry/{ print $2 }' $2)
```
alternatively, we can use pure bash to split the strings to get the consistent results.